core/src/main/java/org/neo4j/ogm/autoindex/AutoIndexManager.java

```diff
@@ -287,7 +299,9 @@ private static Set<AutoIndex> initialiseAutoIndex(MetaData metaData) {
 
                 for (FieldInfo fieldInfo : getIndexFields(classInfo)) {
 
-                    IndexType type = fieldInfo.isConstraint() ? IndexType.UNIQUE_CONSTRAINT : IndexType.SINGLE_INDEX;
+                    IndexType type = fieldInfo.isConstraint() ?
```

Comment:
Sorry, I missed this bit for the ternary operation clean up. Would be good that it gets the same format as the one, you just fixed.

Reply:
Thanks, It's my carelessness. I have fixed it.

Reply:
Happens to be all the time 😄

Reply:
so could you merge this PR now?